HACKING.md

```diff
@@ -776,14 +776,27 @@ not correctly sampled
   * `block` -  will start a block wait time analysis and write `./block.pprof`
   * `web` - start the pprof webserver in process at http://127.0.0.1:6060/debug/pprof
 (you can open this in a browser). This supports `OPENSHIFT_PROFILE_HOST=`
-and `OPENSHIFT_PROFILE_PORT=` to change default ip `127.0.0.1` and default port `6060`.
+and `OPENSHIFT_PROFILE_PORT=` to change default ip `127.0.0.1` and default port `6060`. 
+This works for master components only.
+  * In case of openshift-node, we need to get data from port on which kubelet is running 10250(default port). We can
+   get the pprof data using following command:
+   
+
+   	`curl --cert /tmp/admin.crt --key /tmp/admin.key https://<node_name>:10250/debug/pprof/profile  -k --output /tmp/sample.pprof -L`
```

Comment:
From any admin kubeconfig credentials, you can do:

    $ oc get --raw /debug/pprof/profile --server https://node_name:10250 > /tmp/sample.pprof

which is easier.

Reply:
That is correct and I can update the documentation with both the ways.